### PR TITLE
Update CHANGELOG for 1.57.0

### PR DESCRIPTION
**Pull request by qiwzhang**
Signed-off-by: Wayne Zhang <qiwzhang@google.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# Release 1.57.0 25-05-2021
+===============================================
+
+- update googleapis to the 05/18/2021 (#837)
+
+===============================================
```
Comment (TAOXUY) [inline CHANGELOG.md:6]: - Remove the `==` in line 2 and line 6.
- Capitalize the first word update for consistency.

Comment (qiwzhang) [reply CHANGELOG.md]: Done

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
+# Release 1.57.0 25-05-2021
+
+- update googleapis to the 05/18/2021 (#837)
```
Comment (TAOXUY) [inline CHANGELOG.md:3]: Please capitalize the `update`.

**Review by TAOXUY — DISMISSED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (qiwzhang) [reply CHANGELOG.md]: Done

**Review by nareddyt — APPROVED** [review]

**Pull request merged**